app/resources/views/pages/password_recovery/reset_password.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('layouts.default')
+@section('content')
+<div class="change-password">
+	<h1>Change Password</h1>
+
+	<div class="password-change-form">
+		<form method="post" action="/password-recovery/reset-password">
+			{!! csrf_field() !!}
+			@include('pages.validation_messages', array('errors'=>$errors))
+			<div class="row">
+				<div class="col-xs-12">
```

Comment:
If you just want rows with full 100% width, don't use the Bootstrap grid system because those extra classes and the row element aren't needed.

Remove the &lt;div class="row"&gt; and corresponding close tag.  Also, remove the class="col-xs-12" on those child elements.  The result should look the same but with cleaner code to achieve it.